src/preconnect.js

```diff
@@ -226,10 +227,9 @@ class PreconnectService {
   }
 
   performPreload_(url) {
-    const preload = this.document_.createElement('link');
-    preload.setAttribute('rel', 'preload');
+    const preload = htmlFor(this.document_)`
+        <link rel="preload" referrerpolicy="origin" />`;
     preload.setAttribute('href', url);
```

Comment:
Same question here, is there a reason the url cannot be included here in the tagged template literal?

Reply:
The url is not statically analyzable, so there's not guarantee that it's not malicious.